src/core/a-entity.js

```diff
@@ -500,10 +567,12 @@ var proto = Object.create(ANode.prototype, {
    */
   getAttribute: {
     value: function (attr) {
-      var component = this.components[attr] || components[attr];
-      var value = HTMLElement.prototype.getAttribute.call(this, attr);
-      if (!component || typeof value !== 'string') { return value; }
-      return component.parse(value, true);
+      var component = this.components[attr];
+      // If there's a cached value we just return it
+      if (component && component.attr !== undefined) {
```

Comment:
We should rename `component.attr` to denote that it's a value (`component.value`). `this.components[attr].attr` can be confusing here.